Source/WebKit/WebProcess/GPU/media/SourceBufferPrivateRemote.cpp

```diff
@@ -359,7 +359,7 @@ void SourceBufferPrivateRemote::bufferedSamplesForTrackId(const AtomString& trac
 
 void SourceBufferPrivateRemote::enqueuedSamplesForTrackID(const AtomString& trackId, CompletionHandler<void(Vector<String>&&)>&& completionHandler)
 {
-    if (!m_gpuProcessConnection)
+    if (!isGPURunning())
```

Comment:
Ditto here and maybe elsewhere.

Reply:
indeed, just did a quick search/replace ; didn't pay attention at those existing issues